x-pack/test/apm_api_integration/tests/correlations/latency.spec.ts

```diff
@@ -195,6 +195,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             body: {
               ...getOptions(),
               fieldsToSample: [...fieldsToSample],
+              samplerShardSize: 5000,
```

Comment:
This uses a different size than the runtime code. Should we change this to 10000 and also use `SAMPLER_SHARD_SIZE` from `x-pack/plugins/apm/common/correlations/constants.ts`?